### PR TITLE
Replace user.is_authenticated() with user.is_authenticated for Django 2.0 compatibility

### PR DESCRIPTION
**Pull request by mbaragiola**
Replace user.is_authenticated() with user.is_authenticated for Django 2.0 compatibility

### CODE REVIEW

**Comment by mbaragiola** [comment]
Please note that the tests fail because this change is for Django 2.0, I didn't edit `tox.ini` nor changed tests.

File: rest_framework_extensions/key_constructor/bits.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ class UserKeyBit(KeyBitBase):
     """
 
     def get_data(self, params, view_instance, view_method, request, args, kwargs):
-        if hasattr(request, 'user') and request.user and request.user.is_authenticated():
```
Comment (auvipy) [inline rest_framework_extensions/key_constructor/bits.py]: why request.user is removed?

**Comment by dgilge** [comment]
There are some additional fixes for Django 2.0 in my [repo fork](https://github.com/dgilge/drf-extensions). You might want to take them into account.

**Comment by auvipy** [comment]
hi @dgilge can you help review my pr and give me suggestions about fixing some errors? I picked one of your commit from you branch

**Pull request closed without merging**